apps/livreService/dataService.h

```diff
@@ -43,17 +43,17 @@ class DataService
 {
 public:
     DataService()
-        : _publisher( lunchbox::URI( "livresource://" ))
-        , _subscriber( lunchbox::URI( "livresink://" ))
+        : _publisher( ::zeq::URI( "livresource://" ))
+        , _subscriber( ::zeq::URI( "livresink://" ))
```

Comment:
Yep, I was going to discuss that, before discussion I just put the previous state in zeq::URI language.